content/sql/glossaries/operators/like.md

```diff
@@ -0,0 +1,120 @@
+---
+Title: "SQL Operators: `LIKE`"
+Subjects:
+  - "Language Fluency"
+  - "Code Foundations"
+  - "Data Science"
+Tags:
+  - "Database"
+  - "Queries"
+  - "Tables"
+Catalog Content:
+  - "https://www.codecademy.com/learn/paths/analyze-data-with-sql"
+  - "https://www.codecademy.com/learn/paths/design-databases-with-postgresql"
+---
+
+## Definition
+***
+Allows the user to perform case-sensitive pattern matching on `string` data through the use of wildcard characters.
+
+## Syntax
```

Comment:
This whole section is not very well done. As a reader, I would be pretty confused as to what all the different code blocks even mean. Let's do this:

## Syntax

```sql
SELECT name
FROM movies
WHERE name LIKE 'Star%';
```

The given query will match any movie that begins with `Star` in its title, like `Star Wars` and `Star Trek`.

## `%` Wildcard

The `%` wildcard can be used in a `LIKE` operator pattern to match zero or more unspecified character(s). 

```sql
SELECT name
FROM movies
WHERE name LIKE 'The%';
```

The given query will match any movie that begins with The, followed by zero or more of any characters.

## `_` Wildcard
The `_` wildcard can be used in a `LIKE` operator pattern to match any single unspecified character.

```sql
SELECT name
FROM movies
WHERE name LIKE '_ove';
```

The given query will match any movie which begins with a single character, followed by `ove`.